domain/learningandworkprogress/src/test/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewServiceTest.kt

```diff
@@ -117,4 +122,83 @@ class ReviewServiceTest {
       verify(reviewPersistenceAdapter).getCompletedReviews(PRISON_NUMBER)
     }
   }
+
+  @Nested
+  inner class GetReviewSchedulesForPrisoner {
```

Comment:
We didn't have any unit tests for this method, so I added some whilst I was here